packages/web/src/app/(app)/settings/members/membersTable.tsx

```
@@ -0,0 +1,469 @@
+        currentUserId,
+        activeOwnerCount,
+        hasOrgManagement,
+        scimEnabled,
+    }), [activeOwnerCount, currentUserId, hasOrgManagement, scimEnabled]);
+
+    const handleSortingChange: OnChangeFn<SortingState> = (updater) => {
+        scrollTopBeforeSortRef.current = scrollContainerRef.current?.scrollTop ?? null;
+        setSorting((previous) => functionalUpdate(updater, previous));
+    };
+
+    useLayoutEffect(() => {
```

Biome (2.5.0)

[error] This hook specifies more dependencies than necessary: sorting.

(lint/correctness/useExhaustiveDependencies)